chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,67 @@
+"""DNS A record update.
+
+This file has the function to update the DNS A record
+
+Typical usage example in an experiment json file:
+
+  "module": "chaosgcp.dns.actions",
+  "func": "update_service",
+"""
+
+# Copyright 2023 Google LLC.
```

Comment:
This is an interesting one. We don't usually expliecitely set a copyright, specially at a module level. I'm not against it but we might want to discuss it more extensively.